core/templates/pages/admin-page/services/admin-router.service.ts

```diff
@@ -25,11 +25,18 @@ import { AdminPageConstants } from
 @Injectable({
   providedIn: 'root'
 })
+
 export class AdminRouterService {
   currentTabHash: string = (
     AdminPageConstants.ADMIN_TAB_URLS.ACTIVITIES);
 
+  /**
+   * @param {string} tabHash tab url hash value.
+   * @returns {string|null} tab name of the hash value. If no match then null.
+   */
   getTabNameByHash(tabHash: string): string | null {
+    // Iterates through the ADMIN_TAB_URLS map and returns the
```

Comment:
Descriptions can go above the `@param` etc. parts of the docstring. See https://jsdoc.app/about-getting-started.html for examples.

Reply:
Will change it